curp/src/server/cmd_worker/conflict_checked_mpmc.rs

```diff
@@ -191,9 +209,29 @@ impl<C: Command> Filter<C> {
     fn mark_reset(&mut self, vid: u64) {
         let v = self.get_vertex_mut(vid);
         match v.inner {
-            VertexInner::Reset(ref mut st) => {
-                debug_assert!(matches!(*st, ResetState::Resetting));
-                *st = ResetState::Completed;
+            VertexInner::Reset {
+                ref snapshot,
+                ref finish_tx,
+                ref mut st,
+            } => {
+                debug_assert!(snapshot.is_none(), "snapshot is not taken");
+                debug_assert!(finish_tx.is_none(), "snapshot finish tx is not taken");
+                debug_assert!(matches!(*st, OnceState::Doing));
+                *st = OnceState::Completed;
+            }
+            _ => unreachable!("impossible vertex type"),
```

Comment:
Shall we export the error vertex type in the panic message?

Reply:
Added.